tests/ttnn/unit_tests/operations/test_full_like.py

```diff
@@ -0,0 +1,129 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+
+import torch
+import torch.nn as nn
+import ttnn
+from models.utility_functions import comp_allclose
+from loguru import logger
+
+from tests.ttnn.unit_tests.operations.test_utils import to_npu
+
+
+@pytest.mark.parametrize(
+    "input_shape",
+    [
+        [10],
+        [32, 32],  # single tile
+        [5, 96, 64],  # multiple tiles
+    ],
+)
+@pytest.mark.parametrize(
+    "fill_value",
+    [3, -1],
+)
+@pytest.mark.parametrize(
+    "layout",
+    [
+        ttnn.TILE_LAYOUT,  # Currently only support tile layout
+    ],
+)
+def test_full_like_int(device, input_shape, fill_value, layout):
+    torch_input_tensor = torch.randint(0, 100, (input_shape), dtype=torch.int32)
+    torch_output_tensor = torch.full_like(torch_input_tensor, fill_value)
+
+    input_tensor = ttnn.from_torch(torch_input_tensor, layout=layout)
+    input_tensor = ttnn.to_device(input_tensor, device)
+    output_tensor = ttnn.full_like_2(input_tensor, fill_value)
+    assert ttnn.is_tensor_storage_on_device(output_tensor)
+    output_tensor = ttnn.from_device(output_tensor)
+    output_tensor = ttnn.to_torch(output_tensor)
+    print(input_tensor)
+
+    passing, out = comp_allclose(torch_output_tensor, output_tensor, rtol=0.01, atol=0.01)
```

Comment:
In `full_like`, `assert_equal` seems to be a better choice. Is there still a precision issue?

Reply:
All tests with assert_equal can pass